assets/pages/monthly-contributions/components/paymentMethods.jsx

```diff
@@ -15,22 +16,29 @@ type PropTypes = {
   email: string,
   firstName: string,
   lastName: string,
+  error: ?string,
 };
 
 
 // ----- Component ----- //
 
 function PaymentMethods(props: PropTypes) {
 
-  let content = 'Please fill in all the fields above.';
+  let errorMessage = '';
+  let stripeButton = <StripePopUpButton email={props.email} callback={postCheckout} />;
+  if ((props.firstName === '' || props.lastName === '') && props.error === null) {
```

Comment:
I'm not sure if I'm reading this right, but if one of these fields is blank, and there's an error message, the credit card button will show right? We never want it to show if either of these fields is blank. Would a reasonable alternative be:

```jsx
if ((props.firstName === '' || props.lastName === '') {
  errorMessage = <ErrorMessage message={'Please fill in all the fields above.'} />;
  stripeButton = '';
} else if (props.error !== null) {
  errorMessage = <ErrorMessage message={'There was an error processing your payment. Please\u00a0try\u00a0again\u00a0later.'} />;
}




```